molgenis-data-vcf/src/main/java/org/molgenis/data/vcf/VcfReaderFactoryImpl.java

```diff
@@ -41,6 +44,13 @@ public VcfReader get()
 			if (file.getName().endsWith(".gz"))
 			{
 				inputStream = new GZIPInputStream(inputStream);
+			} 
+			else if (file.getName().endsWith(".zip"))
+			{
+				   ZipFile zipFile = new ZipFile(file.getPath());
+				   Enumeration<? extends ZipEntry> e = zipFile.entries();
+				   ZipEntry entry = (ZipEntry) e.nextElement(); // your only file
```

Comment:
No problem to fix this part. Will push the change soon if needed.
But one concert is that the original fix was focus on format "*.vcf.zip" which mean directly zip original vcf file. For the folder contain vcf file then zip, the extension will be "*.zip". That will be conflicted with csv reader implementation as importWizard can not call the right reader.
Please let me know if I still need to push the change or not.